CONTRIBUTORS

```diff
@@ -470,6 +470,7 @@ Thank you!
     Sergey Merzlikin <sm@smsoft.ru>
     Sergio Durigan Junior <sergiodj@sergiodj.net>
     Sergio Rabellino <rabellino@di.unito.it>
+    Shailesh Vashishth <22538656+vshailesh@users.noreply.github.com>
```

Comment:
This entry is just fine if you like it, but if you prefer to use some other email address, please note that you can adjust this entry accordingly. Our automation is "smart" enough to allow you to pick the email address you prefer...

Reply:
I tried updating to another email, and I am not able to pick another email id. How do I do that?

Reply:
I am working on fixing this. Please wait.

Reply:
@vshailesh, almost done. The current PR branch passes local source maintenance tests, but fails the same tests on GitHub Actions. I know why, but I cannot fix that problem _quickly_.

For now, would you mind changing/setting your GitHub account email to the one you want to see in CONTRIBUTORS (hopefully, that is the same address that this PR branch has listed in CONTRIBUTORS now)? I _hope_ (but cannot easily test) that such a GitHub account update will change the author of the merge commit that GitHub Actions test, resolving the last remaining problem.

We should be handling your use case better, but that will take time to fix... AFAIK, we have not seen it before.